include/mbgl/gfx/types.hpp

```diff
@@ -0,0 +1,124 @@
+#pragma once
+
+#include <mbgl/gfx/gfx_types.hpp>
+
+#include <cstdint>
+
+namespace mbgl {
+namespace gfx {
+
+enum class ColorBlendEquationType : uint8_t {
+    Add,
+    Subtract,
+    ReverseSubtract,
+};
+
+enum class ColorBlendFactorType : uint8_t {
```

Comment:
Suggestion: add a triple slash comment to each enum, describing what it does.

It is also possible to add descriptions to enum values.

````
/// This is an enum class
enum class fooenum {
    FOO, ///< this is foo
    BAR, ///< this is bar
};
````